narwhals/_dask/namespace.py

```diff
@@ -63,6 +63,19 @@ def col(self, *column_names: str) -> DaskExpr:
             backend_version=self._backend_version,
         )
 
+    def lit(self, value: Any, dtype: dtypes.DType | None) -> DaskExpr:
+        # TODO @FBruzzesi: cast to dtype once `reverse_translate_dtype` is implemented.
+        # It should be enough to add `.astype(reverse_translate_dtype(dtype))`
+        return DaskExpr(
+            lambda df: [df._native_dataframe.assign(lit=value).loc[:, "lit"]],
```

Comment:
it's hacky but very easy to maintain the original index

Reply:
works

i'd like to think that dask_expr should be able to optimise this anyway